### DISCUSSION TITLE
telecommunication.json

### DISCUSSION THREAD

**Discussion by waso99** [osmlab/name-suggestion-index#5378] [Q&A] [answered]
Hi
what is the different between "shop/telecommunication.json" & "office/telecommunication.json" under "
name-suggestion-index/data/brands" directory?

Thanks 
waleed

**Comment by bhousel** [comment, marked as answer]
I guess this relates to #5295 and all the previous linked discussion there.

The osm wiki does a pretty good job explaining the difference:
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dtelecommunication
https://wiki.openstreetmap.org/wiki/Tag:office%3Dtelecommunication

and worth reading discussion of why `office=telecommunication` isn't appropriate for retail storefronts:
https://wiki.openstreetmap.org/wiki/Talk:Tag:office%3Dtelecommunication

**Reply by waso99** [reply]
Thanks,
I get it.
